gensim/corpora/dictionary.py

```diff
@@ -24,13 +24,12 @@
 
 from gensim import utils
 
-if sys.version_info[0] >= 3:
-    unicode = str
-
 from six import PY3, iteritems, iterkeys, itervalues, string_types
 from six.moves import xrange
 from six.moves import zip as izip
 
+if sys.version_info[0] >= 3:
```

Comment:
this line should be where it was before.

As per PEP8:

Imports should be grouped in the following order:

standard library imports
related third party imports
local application/library specific imports

You should put a blank line between each group of imports.